room-battle.js

```diff
@@ -232,8 +241,11 @@ class BattleTimer {
 					perTurnTicks = 0;
 				}
 			}
-			this.ticksLeft[slotNum] += this.settings.perTurnTicks;
-			this.turnTicksLeft[slotNum] = Math.min(this.ticksLeft[slotNum], maxTurnTicks);
+			this.battle.room.add(`DEBUG: Dc bank: ${Config.dcTimerBank}, isFirst: ${isFirst}, ticks (slot${slotNum}): ${this.turnTicksLeft[slotNum]}`).update();
+			if (!Config.dcTimerBank || this.turnTicksLeft[slotNum] > 1 || isFirst) {
```

Comment:
I would honestly veto this. I don't want to complicate code by stopping getting time back when you only have the DC bank left.